include/ncengine/graphics/NcGraphics.h

```diff
@@ -95,6 +95,13 @@ struct NcGraphics : public Module
      * is not cleared as it can be set on a persistent Entity.
      */
     virtual void ClearEnvironment() = 0;
+
+    /**
+     * @brief Get the graphics api being used.
+     * @note Typically, this will be equal to GraphicsSettings::preferredApi, but insufficient device support or
```

Comment:
Since this comment will be seen in our public interface will you capitalize API throughout comments in this file?